projects/components/src/file-upload/file-upload.component.ts

```diff
@@ -43,20 +38,46 @@ import { IconSize } from '../icon/icon-size';
           <div class="click-to-upload" (click)="fileInput.click()">Click to upload</div>
           or drag and drop
         </div>
-        <div class="sub-text">{{ this.subText }}</div>
+        <div class="sub-text" *ngIf="this.subText; else defaultSubTpl">{{ this.subText }}</div>
       </div>
 
       <ng-content></ng-content>
     </div>
+
+    <ng-template #defaultSubTpl>
+      <ul class="file-requirements-list">
+        <li class="item">
+          <ht-icon class="icon" icon="${IconType.CheckCircleFill}" size="${IconSize.ExtraSmall}"></ht-icon>
+          <div class="text"><strong>File Types</strong>: {{ this.config.supportedFileTypes | htFilePipe }}</div>
+        </li>
+        <li class="item">
+          <ht-icon class="icon" icon="${IconType.CheckCircleFill}" size="${IconSize.ExtraSmall}"></ht-icon>
+          <div class="text">
+            <strong>Max File Size</strong>: {{ this.config.maxFileSizeInBytes | htDisplayFileSize }}
+          </div>
+        </li>
+        <li class="item">
+          <ht-icon class="icon" icon="${IconType.CheckCircleFill}" size="${IconSize.ExtraSmall}"></ht-icon>
+          <div class="text"><strong>Max Files</strong>: {{ this.config.maxNumberOfFiles | htDisplayNumber }}</div>
+        </li>
+      </ul>
+    </ng-template>
   `
 })
-export class FileUploadComponent implements ControlValueAccessor {
+export class FileUploadComponent {
   @Input()
-  public subText: string = '';
+  public subText?: string;
 
   @Input()
   public disabled: boolean = false;
 
+  @Input()
+  public config: UploaderConfig = {
+    maxNumberOfFiles: 10,
+    maxFileSizeInBytes: 20000000, // 20 MB
```

Comment:
nit: more readable if written as 20 * 1000 * 1000. also implicitly conveys the 20 MB part :)

Reply:
Updated.